Stripe/NSCharacterSet+Stripe.m

```diff
@@ -28,6 +28,24 @@ + (instancetype)stp_invertedAsciiDigitCharacterSet {
     return cs;
 }
 
++ (instancetype)stp_postalCodeCharacterSet {
+    static NSCharacterSet *cs;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        cs = [self characterSetWithCharactersInString:@"abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789- "];
```

Comment:
just to confirm we're confident this shouldn't include, for example, Chinese characters?

Reply:
I did check China (they only use numbers) and have yet to find a country that uses characters outside of this set.